Sources/NIO/BaseSocket.swift

```diff
@@ -334,17 +334,16 @@ class BaseSocket: Selectable {
                 try Posix.bind(descriptor: fd, ptr: ptr, bytes: bytes)
             }
 
+            var addr: SockAddrProtocol
```

Comment:
@weissi I wonder; is this going to force the creation of an existential container? Seems like it might stack allocate one.

Reply:
So I just checked this, and yes, this brings a stack allocated existential into existence. We don't really need the overhead of the extra heap allocation and the associated ref counting/indirection here, so I'm inclined to say that I think this particular change doesn't add much value. Mind removing it?

Reply:
Thanks for checking! Will remove.